vault/logical_system_raft.go

```diff
@@ -375,9 +375,9 @@ func (b *SystemBackend) handleRaftBootstrapAnswerWrite() framework.OperationFunc
 
 		switch nonVoter {
 		case true:
-			err = errors.New("adding non voting peer is not allowed")
+			err = raftBackend.AddPeer(ctx, serverID, clusterAddr, false)
 		default:
-			err = raftBackend.AddPeer(ctx, serverID, clusterAddr)
+			err = raftBackend.AddPeer(ctx, serverID, clusterAddr, true)
```

Comment:
Ahh, right, I see now, `AddPeer` is called here by non-autopilot instances. I gotcha.